.github/workflows/ci.yml

```diff
@@ -213,12 +246,24 @@ jobs:
             plugin/trino-hive-hadoop2/bin/run_hive_alluxio_tests.sh
       - name: Upload test results
         uses: actions/upload-artifact@v2
+        # Upload all test reports only on failure, because the artifacts are large
         if: failure()
         with:
-          name: result hive-tests
+          name: result ${{ github.job }}
           path: |
             **/target/surefire-reports
             **/target/checkstyle-*
+      - name: Upload test report
+        uses: actions/upload-artifact@v2
+        # Always upload the test report for the 'Annotate checks' workflow,
+        # but only the single XML file to keep the artifact small
+        if: always()
+        with:
+          # Name prefix is checked in the `Annotate checks` workflow
```

Comment:
nit: Refer to filename instead of workflow name to make it easier to know where to look.

Reply:
Here and everywhere else.

Reply:
Done!